integration_tests/src/main/python/delta_lake_utils.py

```diff
@@ -157,12 +157,15 @@ def setup_delta_dest_table(spark, path, dest_table_func, use_cdf, partition_colu
     dest_df = dest_table_func(spark)
     writer = dest_df.write.format("delta")
     ddl = schema_to_ddl(spark, dest_df.schema)
-    table_properties = {}
-    if use_cdf:
-        table_properties['delta.enableChangeDataFeed'] = 'true'
-    if enable_deletion_vectors:
-        table_properties['delta.enableDeletionVectors'] = 'true'
-    if len(table_properties) > 0:
+    table_properties = {
+        'delta.enableChangeDataFeed': str(use_cdf).lower(),
+    }
+    # prevent on 11.3: Unknown configuration was specified: delta.enableDeletionVectors
+    # TODO OSS Delta 2.3+
+    if is_databricks122_or_later():
+        table_properties['delta.enableDeletionVectors'] = str(enable_deletion_vectors).lower()
```

Comment:
So we only enable deletion vectors if we are on databricks 122 or later? I am not super familiar with how this code is used, but if someone asks to enable a feature and we ignore it, that feels bad to me.

Like for the test 

https://github.com/NVIDIA/spark-rapids/blob/b92366d3607f814ae563378076bd08cd08b5c83f/integration_tests/src/main/python/delta_lake_test.py#L77

Which is supposed to fall back because deletion vectors are enabled. If we don't enable then unconditionally are we not testing the right thing?

Reply:
This is just to prevent AnalysisException on pre-12.2 which does not support deletion vectors. The test you point out uses an idiom `supports_delta_lake_deletion_vectors` capturing what I needed but did not realize exists, updating

Reply:
We cannot enable_deleltion_vectors before 12.2, hence all tests where enable_deletion_vectors may be True are 
```
@pytest.mark.skipif(not supports_delta_lake_deletion_vectors(), reason="Deletion vectors are new in Spark 3.4.0 / DBR 12.2")
```